_posts/2016/2016-04-21-jenadevs-docker-party.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "First Jena Devlopers Meetup: Docker Party!"
+date: "2016-04-11 13:37:42"
+icon: cubes
+categories: events
+authors: ["Benjamin and Oliver Z."]
+---
+
+On the first Thursday of April the newly founded [Softwerkskammer Jena](https://www.softwerkskammer.org/groups/jena) had its first [Developers Meetup](www.meetup.com/jenadevs).
```

Comment:
Change to: 
Beginning of April, ePages hosted the first Developer Meetup of Softwerkskammer Jena.